src/app/pages/component-sidenav/component-sidenav.scss

```diff
@@ -104,7 +104,7 @@ div.docs-component-viewer-nav-content .mat-nav-list .mat-mdc-list-item .mat-list
 
 @media (max-width: 720px) {
   .docs-component-viewer-sidenav-container {
-    flex: 1 0 auto;
+    flex: 1 0;
```

Comment:
This file doesn't exist anymore in the latest main branch, and the issue was resolved on version 18.0.3